app/views/components/_tabs.html.erb

```diff
@@ -0,0 +1,24 @@
+<%
```

Comment:
Is this "tabs" component something that could end up in govuk_publishing_components?
Can you add some tests for it?

Reply:
Tests added